python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_lognormal.py

```diff
@@ -0,0 +1,293 @@
+deg_list = [5, 10, 20, 30, 40]
+g = ot.Graph('Polynomials orthonormal  with respect to the LogNormal distribution', 'x', 'y', True)
+leg = ot.Description(0)
+for indice, deg in enumerate(deg_list):
+    coeff_Poly_Deg = sample_Coeff[0:deg]
+    poly_Deg = ot.OrthogonalUniVariatePolynomial(coeff_Poly_Deg)
+    g.add(poly_Deg.draw(lower_Bound[0], upper_Bound[0], 1024))
+    leg.add(r'$d = $' + str(deg))
+    print('degree = ', deg, 'polynomial(0) = ', poly_Deg(0))
+
+g.setLegends(leg)
+g.setLegendPosition('topleft')
+view = otv.View(g)
+
+# %%
+# We observe that the polynomials up to degree 40:
+#
+# - are computed in a stable manner, as indicated by their smooth graphs,
+# - take reasonable values on the interval :math:`[0, 25]`.
+
+# %%
+# Case 1: PCE using orthonormal polynomial family w.r.t. the LogNormal distribution
+# ---------------------------------------------------------------------------------
+#
+# We define :math:`\model`, the input random variable :math:`X` and the output random variable :math:`Y = \model(X)`.
+input_Dim = dist_X.getDimension()
+input_RV = ot.RandomVector(dist_X)
+
+g = ot.SymbolicFunction('x', '1/(x)')
+
+output_RV = ot.CompositeRandomVector(g, input_RV)
+
+# %%
+# We create a training sample.
+sample_Size = 1000
+input_Train = dist_X.getSample(sample_Size)
+output_Train = g(input_Train)
+print('max sample = ', input_Train.getMax()[0])
+
+
+# %%
+# We now construct the polynomial chaos surrogate model :math:`\metaModel`
+# using the polynomials up to degree 5. Note that the coefficients in the linear combination of
+# the polynomials are large.
+basis_LN = ot.OrthogonalProductPolynomialFactory([ot.StandardDistributionPolynomialFactory(ot.LogNormal())])
+basis_Size_LN = 6
+chaos_Algo_LN = ot.LeastSquaresExpansion(input_Train, output_Train, dist_X, basis_LN, basis_Size_LN)
+chaos_Algo_LN.run()
+result_LN = chaos_Algo_LN.getResult()
+meta_Model_LN = result_LN.getMetaModel()
+print('meta_Model_LN = ', meta_Model_LN)
+print('coeff = ', result_LN.getCoefficients())
+
+# %%
+# We assess the quality of the surrogate model using the :class:`~openturns.MetaModelValidation` class.
+# We compute the :math:`R^2` score, which is very low.
+meta_Model_Predictions_LN = meta_Model_LN(input_Train)
+valid_Meta_Model_LN = ot.MetaModelValidation(output_Train, meta_Model_Predictions_LN)
+r2_Score_LN = valid_Meta_Model_LN.computeR2Score()[0]
+r2_Score_LN
+
+graph_Valid_LN = valid_Meta_Model_LN.drawValidation()
+graph_Valid_LN.setTitle(r"$R^2=$%.2f%%, LogNormal polyn. family, $d \leq $" % (r2_Score_LN * 100) + str(basis_Size_LN - 1))
+view = otv.View(graph_Valid_LN)
+
+# %%
+# Here, we plot the model and the surrogate model.
+# The differences in the region where :math:`x \rightarrow +\infty` are not significant,
+# since the LogNormal distribution assigns negligible probability mass to this region.
+# Thus, even if the surrogate model deviates from the true model for :math:`x \rightarrow +\infty`,
```

Comment:
This sentence may be confusing since we do not have any picture of the particular LogNormal distribution at hand. This is why I suggest to plot its PDF.

Reply:
ok..; I add it.